frontend/src/components/CharacterList/CharacterList.tsx

```diff
@@ -1,32 +1,46 @@
 import React from 'react';
-import { SaveGameDto, HeroDto } from '../../types/types.ts';
-import { SaveGameCard } from '../SaveGameCard/SaveGameCard.tsx';
+import { CharacterListProps } from '../../types/types';
+import DeleteSaveGame from '../DeleteSaveGame/DeleteSaveGame';
+import { Link, useLocation } from 'react-router-dom';
 import './CharacterList.css';
+import './../../App.css';
 
-interface CharacterListProps {
-    saveGames: SaveGameDto[];
-    heroes: HeroDto[];
-    onDeleteSaveGame: (saveId: string) => void;
-    onUpdateHeroClick: (hero: HeroDto | undefined) => void;
-}
+const CharacterList: React.FC<CharacterListProps> = ({
+                                                         saveGames,
+                                                         heroes,
+                                                         onDeleteSaveGame,
+                                                         onDeleteHero,
+                                                         onUpdateHeroClick,
+                                                     }) => {
+    const location = useLocation();
 
-const CharacterList: React.FC<CharacterListProps> = ({ saveGames, heroes, onDeleteSaveGame, onUpdateHeroClick }) => {
     return (
-        <div className="CharacterListContainer">
+        <section className="CharacterListContainer">
             {saveGames.map(saveGame => {
-                const hero = heroes.find(hero => hero.id === saveGame.heroId);
+                const hero = heroes.find(h => h.id === saveGame.heroId);
+
                 return (
-                    <SaveGameCard
-                        key={saveGame.saveId}
-                        saveGame={saveGame}
-                        hero={hero}
-                        onDeleteSaveGame={onDeleteSaveGame}
-                        onUpdateHeroClick={() => onUpdateHeroClick(hero)}
-                    />
+                    <div className="save-game-card-container" key={saveGame.saveId}>
+                        <p className="hero-name">Name: {hero?.name ?? 'Unknown'}</p>
+                        <p className="location">Location: {location.pathname}</p> {/* Use location.pathname */}
+                        <DeleteSaveGame
+                            saveGame={saveGame}
+                            onDeleteSaveGame={() => onDeleteSaveGame(saveGame.saveId)}
+                            onDeleteHero={() => onDeleteHero(hero?.id ?? '')}
+                        />
+                        <button className="update-hero-button" onClick={() => onUpdateHeroClick(hero)}>
+                            Update Hero
+                        </button>
+                        <Link to={location.pathname === '/character-selection' ? '/start' : location.pathname}>
+                            <button className="start-button">
+                                Start
+                            </button>
+                        </Link>
+                    </div>
```

Comment:
You're calling your component `......List.....` but no list is to find here?`Just a number of divs. For semantical purpose, I highly recommend using a list here for the save games instead of divs.